File: telephoneGame.php
<?php
function telephoneGame($m) {
    $a = -1;
    for($i=1;$i<=(count($m)-1);$i++)
    {
        if ($m[$i]!==($m[0])){
        $a = $i;
        break;
        }
    }
    return $a;
}
?>

File: telephoneGame.java
int telephoneGame(String[] m) {
  int a = -1;
  for (int i = 1; i < m.length; i++) {
    if (!m[i].equals(m[i - 1])) {
      a = i;
      break;
    }  
  }
  return a;
}
